dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/support/AccessLogData.java

```diff
@@ -239,7 +240,7 @@ public String getLogMessage() {
 
         Object[] args = get(ARGUMENTS) != null ? (Object[]) get(ARGUMENTS) : null;
         if (args != null && args.length > 0) {
-            sn.append(JSON.toJSONString(args));
+            sn.append(JSON.toJSONString(args, SerializerFeature.IgnoreErrorGetter));
```

Comment:
Ignoring this property is not the best way. It is recommended to print an exception once to solve the problem of continuously filling the queue.